websites/M/Modrinth/presence.ts

```diff
@@ -0,0 +1,210 @@
+const presence = new Presence({
+		clientId: "1047217921685991462",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/GnMSCG4.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, hostname, href } = document.location,
+		pathSplit = pathname.split("/").filter(x => x),
+		active = document.querySelector('[class*="is-active"]');
+
+	switch (hostname) {
+		case "modrinth.com": {
+			switch (pathSplit[0]) {
+				case "user": {
+					presenceData.details = `Browsing ${pathSplit[1]}'s profile`;
+					presenceData.buttons = [
+						{
+							label: "View Profile",
+							url: href,
+						},
+					];
+					break;
+				}
+				case "settings": {
+					presenceData.details = "Browsing profile settings";
+					break;
+				}
+				case "mods": {
+					presenceData.details = "Looking for wonderful mods";
+					break;
+				}
+				case "mod": {
+					presenceData.state = `${
+						document.querySelector<HTMLHeadingElement>("h1.title").textContent
+					}`;
+					presenceData.buttons = [
+						{
+							label: "View Mod",
+							url: href,
+						},
+					];
+					if (active?.textContent) {
+						presenceData.details = `Browsing ${
+							pathSplit[0]
+						} ${active?.textContent.toLowerCase()}`;
+						presence.setActivity(presenceData);
+						return;
+					}
+					break;
+				}
+				case "plugins": {
+					presenceData.details = "Looking for useful plugins";
+					break;
+				}
+				case "plugin": {
+					presenceData.state = `${
+						document.querySelector<HTMLHeadingElement>("h1.title").textContent
+					}`;
+					presenceData.buttons = [
+						{
+							label: "View Plugin",
+							url: href,
+						},
+					];
+					if (active?.textContent) {
+						presenceData.details = `Browsing ${
+							pathSplit[0]
+						} ${active?.textContent.toLowerCase()}`;
+						presence.setActivity(presenceData);
+						return;
+					}
+					break;
+				}
+				case "resourcepacks": {
+					presenceData.details = "Looking for a beautiful resourcepack";
+					break;
+				}
+				case "resourcepack": {
+					presenceData.state = `${
+						document.querySelector<HTMLHeadingElement>("h1.title").textContent
+					}`;
+					presenceData.buttons = [
+						{
+							label: "View Resourcepack",
+							url: href,
+						},
+					];
+					if (active?.textContent) {
+						presenceData.details = `Browsing ${
+							pathSplit[0]
+						} ${active?.textContent.toLowerCase()}`;
+						presence.setActivity(presenceData);
+						return;
+					}
+					break;
+				}
+				case "modpacks": {
+					presenceData.details = "Looking for a nice modpack";
+					break;
+				}
+				case "modpack": {
+					presenceData.state = `${
+						document.querySelector<HTMLHeadingElement>("h1.title").textContent
+					}`;
+					presenceData.buttons = [
+						{
+							label: "View Modpack",
```

Comment:
```suggestion
							label: "View Mod Pack",
```

Reply:
on the morinth website, the word modpack is written without separation